sw/host/provisioning/orchestrator/src/ot_dut.py

```diff
@@ -115,11 +114,37 @@ def run_cp(self) -> None:
         # Run provisioning flow and collect logs.
         res = run(cmd, f"{self.log_dir}/cp_out.log.txt",
                   f"{self.log_dir}/cp_err.log.txt")
```

Comment:
We can probably stage the run in a tempdir, and then move the files to expected {device_id} folder after execution. wdyt?

Reply:
yep that would probably be cleaner and simplify retries.

Reply:
Done.